### PR TITLE
Update release notes todos section and add debugging tips

### PR DESCRIPTION
**Pull request by npolshakova**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

Adds a common debug section and updates the TODOs section based on 1.30 improvement discussions: https://kubernetes.slack.com/archives/CN1KH4K9A/p1706942615321299?thread_ts=1706900606.161789&cid=CN1KH4K9A 

#### Which issue(s) this PR fixes:
https://github.com/kubernetes/sig-release/issues/2498

### CODE REVIEW

File: release-team/role-handbooks/release-notes/README.md [hunk 1]
```diff
@@ -214,11 +214,13 @@ Begin running release-notes tool for the ongoing collection of release notes wit
 - __Release day__
 - Copy notes from Google Doc to HackMD in markdown
 - Final version of release notes committed for release
+- Close the _Known Issues_ Issue is and make sure everything has been resolved
```
Comment (ramrodo) [inline release-team/role-handbooks/release-notes/README.md:217]:
```suggestion
- Close the _Known Issues_ Issue and make sure everything has been resolved
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
Thanks, Nina.

Just a small suggestion for a typo.

File: release-team/role-handbooks/release-notes/README.md [hunk 2]
```diff
@@ -214,11 +214,13 @@ Begin running release-notes tool for the ongoing collection of release notes wit
...
 - Release Notes must be merged into master prior to the release. If this is not done the release will include the latest draft.
 
 ### Week 19
 
 - Release retrospective participation
+- Update the TODOs section at the end of each release for the next Release Notes Team
```
Comment (rashansmith) [inline release-team/role-handbooks/release-notes/README.md:223]: nice!

File: release-team/role-handbooks/release-notes/README.md [hunk 3]
```diff
@@ -230,17 +232,35 @@ Begin running release-notes tool for the ongoing collection of release notes wit
 - [LWKD](http://lwkd.info) *(consider contributing to LWKD as part of your role)*
 - [Kubernetes Documentation Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/)
 
+## Debugging Tips
+
+If you are having trouble running the `krel` tool, here are some common issues and solutions:
+
+1. Try running with `--log-level=debug` or `--log-level=trace` to get more information about what is going wrong.
```
Comment (rashansmith) [inline release-team/role-handbooks/release-notes/README.md:239]: Can also mention the path to the /var/k8s.... . Overall a good explanation of the file structure of Release Notes and how its updated between the maps and yaml files could be helpful as well.

File: release-team/role-handbooks/release-notes/README.md [hunk 4]
```diff
@@ -230,17 +232,35 @@ Begin running release-notes tool for the ongoing collection of release notes wit
...
+areas of improvement:
 
-### Post 1.17 TODOs
+#### Github Workflow to Detect Common Release Note Issues
 
-#### If any team members have NLP experience
+- yaml linter to block invalid yaml merging in from manually edited release notes
+- spell check based on dictionary of common Kubernetes terms
+- check for correct punctuation in release notes
+- check for incorrect tense in release notes
```
Comment (rashansmith) [inline release-team/role-handbooks/release-notes/README.md:253]: Do you have a link to the Github actions that was worked on in 1.30?

File: release-team/role-handbooks/release-notes/README.md (hunk 2 above)
Comment (rashansmith) [reply release-team/role-handbooks/release-notes/README.md:223]: Also would want to update other things like tool documentation, debugging tips, etc

File: release-team/role-handbooks/release-notes/README.md (hunk 4 above)
Comment (npolshakova) [reply release-team/role-handbooks/release-notes/README.md:253]: Good idea! I'll link it in this section!

**Review by rashansmith — APPROVED** [review]

**Pull request merged**